CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 ## Version 0.25.0 - TBD
 ### Added
 - Declaring a type alias on an enum in cds now also exports it on value level in the resulting type
+- Apps need to provide `@sap/cds` version `8.2` or higher.
+- Apps need to provide `@cap-js/cds-types` version `0.6.4` or higher.
```

Comment:
I guess the feature itself also deserves an entry here 😃

Reply:
Done